migration/src/test/java/org/axonframework/migration/AxonJakartaTest.java

```diff
@@ -0,0 +1,79 @@
+package org.axonframework.migration;
```

Comment:
I'm missing the copyright notice in this file. You can find the notice template [here](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml)

Reply:
Applied; Although it's also missing in for instance `OnlyAcceptConstructorPropertiesAnnotation`. 

You might be interested in fixing that in bulk through [AddApache2LicenseHeader](https://public.moderne.io/recipes/org.openrewrite.java.AddApache2LicenseHeader). 

Preview & pull request link here: https://public.moderne.io/results/VBLvS